core.html

```diff
@@ -288,24 +288,24 @@ <h3>Method 3: DID Shortening with SHA-256 Hash</h3>
                 DID
                 identifier derived from a SHA-256 hash of the original identifier. The benefit of using Method 3 over Method to is the ability to have smaller size didcomm messages as did:peer2. dids tend to be verbose in nature. Method 3 peer dids can only be used after a peer did method 2 has been exchange with the other party and thus can map the shortened did to the longform one. In order to send a message encrypted with method 3 you first MUST send a discover-feature message (using the method 2 as the `to` field) to make sure that the receiving agent is capable of resolving method 3 dids.</p>
             <pre class="example" class="ABNF" title="ABNF for peer DIDs">
-peer-did-method-3 = "did:peer:3." 64*HEXDIG
+peer-did-method-3 = "did:peer:3" 64*HEXDIG
```

Comment:
I think we want something more along the lines of:

```suggestion
peer-did-method-3 = "did:peer:3" transform encnumbasis
transform = "z"
encnumbasis = 46*BASE58BTC
```